renderer/src/main.cpp

```diff
@@ -10,10 +10,115 @@
 
 #include <iostream>
 #include <chrono>
+#include <memory>
+
+#include <boost/log/core.hpp>
```

Comment:
Well, these should be relocated into separate library, along with setup function